examples/cov2d_simulation.py

```diff
@@ -54,20 +55,19 @@
 filters = [RadialCTFFilter(pixel_size, voltage, defocus=d, Cs=2.0, alpha=0.1)
            for d in np.linspace(defocus_min, defocus_max, defocus_ct)]
 
-# Create a simulation object with specified filters
+# Create a simulation object with specified filters and 3D map of a 70S Ribosome
+logger.info('Load 3D map and creat simulation object.')
 sim = Simulation(
     n=num_imgs,
+    vols_file=os.path.join(DATA_DIR, 'clean70SRibosome_vol_65p.mrc'),
```

Comment:
Why not give the volumes themselves as a parameter?

Reply:
Also, this file is huge. Do we have to store it in the repository?

Reply:
Try to set an example so that other system maps can be used too. The volume file is a downsampled one, only 1M. It is relative small compared to other files in the src/data folder.

Reply:
Changed to initialize a simulation object using volumes in new commit 949858b. Created the issue #104 for the size problem of input data.